testsuite/features/support/client_stack.rb

```diff
@@ -67,17 +67,24 @@ def check_restart(host, node, time_out)
   end
 end
 
-# Extract the OS version by decoding the value in '/etc/os-release'
-# Use VERSION id = false, VERSION_ID otherwise
-def get_os_version(node, id = true)
-  field = id ? 'VERSION_ID' : 'VERSION'
-  os_version_raw, _code = node.run('grep "' + field + '=" /etc/os-release')
+# Extract the OS version and OS family
+# We get the OS version by decoding the value in '/etc/os-release'
+# We use VERSION for SLE and VERSION_ID for openSUSE
```

Comment:
While this works, just a word of warning: `VERSION` should not be used by scripts (`man os-release`):
```
       VERSION=
           A string identifying the operating system version, excluding any OS name information, possibly including a release code name, and suitable for presentation to the user. This field is optional. Example: "VERSION=17" or "VERSION="17 (Beefy Miracle)"".

       VERSION_ID=
           A lower-case string (mostly numeric, no spaces or other characters outside of 0–9, a–z, ".", "_" and "-") identifying the operating system version, excluding any OS name information or release code name, and suitable for processing by scripts or usage
           in generated filenames. This field is optional. Example: "VERSION_ID=17" or "VERSION_ID=11.04".
```

Reply:
Could we extract the version of an SLE OS, using VERSION_ID?
If it was the case, I would consider to always use VERSION_ID.

Reply:
As for SLE `VERSION_ID` is `X.Y`, we'd need a transformation to `X-SPY`. That would allow always using `VERSION_ID`.

Reply:
Yes, and also, what would be the benefit of that?